native/c/makefile

```diff
@@ -94,13 +40,12 @@ CPP_FLAGS+=$(DEBUGGER_FLAGS) $(OTHER_C_FLAGS)
 MTL_FLAGS+=$(DEBUGGER_FLAGS) $(OTHER_C_FLAGS)
 MTL_FLAGS64+=$(DEBUGGER_FLAGS) $(OTHER_C_FLAGS)
 ASM_FLAGS+=--verbose
-
 .ELSE
 C_FLAGS+=-g0
 CPP_FLAGS+=-g0
-CPP_BND_FLAGS+=-g0
-CPP_BND_FLAGS_AUTH+=-g0
-DLL_BND_FLAGS+=-g0
+CPP_BND_FLAGS=$(CPP_BND_BASE_FLAGS)
+CPP_BND_FLAGS_AUTH=$(CPP_BND_BASE_FLAGS_AUTH)
+DLL_BND_FLAGS=$(DLL_BND_BASE_FLAGS)
```

Comment:
Thanks for considering the feedback and cleaning up this Makefile 🙏 
I believe we don't need to reset these 3 variables though 😅

Reply:
> they are initially assigned to the debug flags on lines 28-30 and reassigned to the base flags in the .ELSE block if the build type is not debug

Is it necessary to reassign them to the same values when the build type is not debug?

Either way, it should be fine 😅